.github/scripts/generate_binary_build_matrix.py

```diff
@@ -154,41 +154,28 @@ def arch_type(arch_version: str) -> str:
         return "cpu"
 
 
-# This can be updated to the release version when cutting release branch, i.e. 2.1
 DEFAULT_TAG = os.getenv("RELEASE_VERSION_TAG", "main")
 
 WHEEL_CONTAINER_IMAGES = {
+    **{gpu_arch: f"manylinux2_28-builder:cuda{gpu_arch}" for gpu_arch in CUDA_ARCHES},
```

Comment:
```suggestion
    **{gpu_arch: f"wheel-builder:cuda{gpu_arch}" for gpu_arch in CUDA_ARCHES},
```

nit: but can we just name these something that's easier to type

Reply:
When we are going to update to next wheel build platform I think we would need to keep the old wheels and introduce the new ones. Hence having manylinux version as in pypi package comes handy